File: test/ql/qltest.php
<?php

$db = new mysqli('0', '', '', 'testdb', 9306);
$result = $db->query('SELECT * FROM rt');
$row = $result->fetch_row();
printf("%s | %s | %s | %s\n", $row[0], $row[1], $row[2], $row[3]);

?>

File: test/ql/qltest.rb
require "mysql"

db = Mysql.real_connect('0', '', '', 'testdb', 9306)
res = db.query('SELECT * FROM rt')
row = res.fetch_row
puts row[0] + ' | ' + row[1] + ' | ' + row[2] + ' | ' + row[3];
